src/Models/Constants/Types.ts

```diff
@@ -171,6 +171,15 @@ export type CardboardIconNames =
     | 'SpeedHigh'
     | 'View';
 
+export enum DurationUnits {
```

Comment:
I think you set the values for enums (0,1,2,3,4,5) to make comparison in another file (`formatTimeInRelevantUnits` method), but still we don't need to set the values for enum keys as you can still make < > comparisons

Reply:
no downside, just the values being a number for "DurationUnits" might be confusing